docs/contributing/forem.md

```diff
@@ -0,0 +1,180 @@
+mentioning us on the issue.
+
+**Documentation** is almost always a great place to start contributing to a new
+project. DEV is an Open Source, community-driven project. Therefore, providing
+and maintaining quality documentation is one of our most important jobs. You can
+find more information in our
+[docs guide](https://docs.dev.to/contributing/docs)!
+
+**Refactoring** (improving the code without modifying behavior) is a great place
+to help out! Generally speaking, you can rely on existing tests to ensure that
+your refactor doesn't introduce any unexpected behavior. If an area isn't well
+tested, you might be asked to include a regression test with your refactoring
+PR. Refactors can touch many files, so we encourage breaking big changes into
+small PRs.
+
+**Fixing bugs** is a super fast way to improve the experience for our users!
+When you're fixing bugs, we appreciate communication in a GitHub issue. If an
+issue exists, please claim that issue and link it in your PR, otherwise creating
+an issue is the best first step! Be sure to surround bug fixes with ample tests;
+bugs are magnets for other bugs. Write tests around bugs!
+
+**Building features** requires a lot of communication, but we'd love to have
+your help with this too! Features tend to be subjective and might spur some
+debate. Be sure to create an issue for new features before getting started! If
+your feature involves design changes, including design mockups can be very
+helpful. As always, when in doubt, ask!
+
+### How to contribute
+
+1. [Fork the project](https://docs.dev.to/getting-started/forking/) and clone it
+   to your local machine. Follow the
+   [installation guide](https://docs.dev.to/installation/)!
+2. Create a branch with your GitHub username and the ID of the
+   [issue](https://github.com/thepracticaldev/dev.to/issues), for example:
+   `git checkout -b USERNAME/some-new-feature-1234`
+3. Code and commit your changes. Bonus points if you write a
+   [good commit message](https://chris.beams.io/posts/git-commit/):
+   `git commit -m 'Add some feature'`
+4. Push to the branch: `git push -u origin USERNAME/some-new-feature-1234`
+5. [Create a pull request](https://docs.dev.to/getting-started/pull-request/)
+   for your branch. 🎉
+
+## Contribution guidelines
+
+### Create an issue
+
+Nobody's perfect. Something doesn't work? Something could be better? Check to
+see if the issue already exists, and if it does, leave a comment to get our
+attention! If the issue doesn't already exist, feel free to create a new one. A
+core team member will triage incoming issues.
+
+_Please note: core team members may update the title of an issue to reflect the
+discussion._
+
+### Please include tests
+
+Some areas of the project could use updated tests, and new features should
+always include test coverage. Please give our
+[testing guide](https://docs.dev.to/tests/) a read!
+
+### Code quality
+
+We use [CodeClimate](https://codeclimate.com/) to find code smells. If a pull
```

Comment:
```suggestion
We use [Code Climate](https://codeclimate.com/) to find code smells. If a pull
```
Did a quick grep through their site and they refer to themselves with a space between